parity/cli/mod.rs

```diff
@@ -238,6 +238,11 @@ usage! {
 	{
 		// Global flags and arguments
 		["Operating Options"]
+			FLAG flag_allow_empty_block_result: (bool) = false, or |c: &Config| c.parity.as_ref()?.allow_empty_block_result
+			.clone(),
+			"--no-empty-block-result",
```

Comment:
I think this flag should be renamed to `allow-empty-block-result` since enabling it allows empty block results (instead of disallowing).